src/route.ts

```diff
@@ -41,6 +41,15 @@ export default (store: Store<State> & LazyStore) => {
     @property({ type: String })
     private path?: string;
 
+    @property({ type: Boolean })
+    private isResolving = false;
+
+    @property({ type: Function })
+    private resolve;
```

Comment:
This one should also be optional, and needs a type
```
private resolve?: () => Promise<any>;
```

Reply:
eslint threw https://palantir.github.io/tslint/rules/no-any/ just went with {}